x-pack/plugins/uptime/public/components/common/uptime_date_picker.test.tsx

```diff
@@ -12,8 +12,13 @@ import { createMemoryHistory } from 'history';
 import { render } from '../../lib/helper/rtl_helpers';
 import { fireEvent } from '@testing-library/dom';
 
-// FLAKY: https://github.com/elastic/kibana/issues/114396
-describe.skip('UptimeDatePicker component', () => {
+describe('UptimeDatePicker component', () => {
+  jest.setTimeout(10_000);
```

Comment:
```suggestion
  jest.setTimeout(10_000);
```

Always love seeing that syntax 🥇